### DISCUSSION TITLE
养生壶无法控制

### DISCUSSION THREAD

**Discussion by realxmtx** [XiaoMi/ha_xiaomi_home#441] [Ideas]
![image](https://github.com/user-attachments/assets/0f11b8cf-f722-47f6-b3a1-cf15db3c0964)

如上图所示，米家养生壶只能显示设备当前状态或停止烹饪，但不能选择烹饪（比如烧水等）。使用miot集成则正常。

**Comment by SusanPhevos** [comment]
同 #348
